packages/server/utils/makeSentryRequest.ts

```diff
@@ -0,0 +1,18 @@
+import uwsGetHeaders from './uwsGetHeaders'
+import getReqAuth from './getReqAuth'
+import uwsGetIP from './uwsGetIP'
+import {HttpResponse, HttpRequest} from 'uWebSockets.js'
+
+const makeSentryRequest = (res: HttpResponse, req: HttpRequest) => {
```

Comment:
never ended up using this, but keeping this in case we do in the future. this is everything sentry thinks exists on a request object